InomialClient.js

```diff
@@ -7,12 +7,19 @@
 // subscriptions.
 //
 
+// Log level constants
+const NONE = 0;     // No logging at all.
```

Comment:
please remove these log levels. If you want to include a VERBOSE mode that prints requests/response and other stuff then that's OK. Otherwise the client should print errors only. But we don't need 4 logging levels.